<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConsoleLog</class>
 <widget class="QWidget" name="ConsoleLog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Console Log</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>450</width>
       <height>91</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QToolButton" name="saveLogBtn">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>Art_Assets/saveIcon.bmp</normaloff>Art_Assets/saveIcon.bmp</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="shortcut">
       <string/>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
      <property name="autoRaise">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="clearLogBtn">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>0</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear Log</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>Art_Assets/clearLogIcon.png</normaloff>Art_Assets/clearLogIcon.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="shortcut">
       <string/>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
      <property name="autoRaise">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="printLogBtn">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>0</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Print</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>Art_Assets/printIcon.png</normaloff>Art_Assets/printIcon.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="shortcut">
       <string/>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
      <property name="autoRaise">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="optionBtn">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>0</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Options</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>Art_Assets/optionIcon.png</normaloff>Art_Assets/optionIcon.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="shortcut">
       <string/>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
      <property name="autoRaise">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="searchLogBtn">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>0</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>Art_Assets/searchIcon.png</normaloff>Art_Assets/searchIcon.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="shortcut">
       <string/>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
      <property name="autoRaise">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTextBrowser" name="consoleLogTxt">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>400</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
